galaxyui/src/app/app-routing.module.ts

```diff
@@ -14,13 +14,11 @@ import { NamespaceDetailResolver, RepositoryResolver as AuthorRepositoryResolver
 import { PreloadAllModules, RouterModule, Routes } from '@angular/router';
 
 const appRoutes: Routes = [
+    // Lazily loaded modules
     {
-        path: '',
-        redirectTo: '/home',
-        pathMatch: 'full',
+        path: 'home',
+        loadChildren: './home/home.module#HomeModule',
```

Comment:
we don't need to lazy load home. Just deleting the redirect here will be enough.